test/Interop/Cxx/destructor/basic.swift

```diff
@@ -0,0 +1,61 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop)
```

Comment:
Could you rename this test and other tests to follow the naming pattern of other tests in `test/Interop`? `basic-typechecker.swift`, `basic-irgen.swift` etc.?